aws/data_source_aws_mq_broker_test.go

```diff
@@ -0,0 +1,169 @@
+package aws
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform/helper/acctest"
+	"github.com/hashicorp/terraform/helper/resource"
+)
+
+func TestAccDataSourceAWSMqBroker_basic(t *testing.T) {
```

Comment:
```
=== RUN   TestAccDataSourceAWSMqBroker_basic
--- FAIL: TestAccDataSourceAWSMqBroker_basic (1300.05s)
    testing.go:518: Step 0 error: Check failed: 2 error(s) occurred:
        
        * Check 3/28 error: data.aws_mq_broker.by_id: Attribute 'configuration.#' expected "2", got "1"
        * Check 17/28 error: data.aws_mq_broker.by_name: Attribute 'configuration.#' expected "2", got "1"
```

FYI you might also want to check out `resource.TestCheckResourceAttrPair()` to compare the resource attributes to the data source attributes, rather than trying to figure out values here.

Reply:
Nice 👍 I saw it once or twice but didn't check what it really does.